lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -50,6 +69,9 @@ class TestUserEnrollmentApi(MobileAPITestCase, MobileAuthUserTestMixin, MobileCo
     REVERSE_INFO = {'name': 'courseenrollment-detail', 'params': ['username']}
     ALLOW_ACCESS_TO_UNRELEASED_COURSE = True
     ALLOW_ACCESS_TO_MILESTONE_COURSE = True
+    TOMORROW = datetime.datetime.now(pytz.UTC) + datetime.timedelta(days=1)
```

Comment:
On second thought, that probably wouldn't cause them to fail, because we're dealing with `datetimes`, not `dates`. It might be good to change it `NEXT_WEEK` anyway, though, just in case any access checking code somewhere in the platform decides to check against dates instead of datetimes.